### PR TITLE
Hardhat Deployment scripts + new contract names in MythX config

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

**New deployment scripts** to enable to deploy one or multiple contracts on target network, using one CLI command 🚀 

These deployment scripts were written using hardhat-deploy community plugin for:
- [x] hardhat local network
- [x] L14 Network ⛓️ 
- [x] ensure Base Contracts (especially `UniversalProfileInit`) are initialized (calling the function `initialize(zeroAddress)` after deployment 🔒 
- [x] Edit contract names + paths in MythX `.yml` config file

### CODE REVIEW

**Comment by frozeman** [comment]
we are missing the README instructions

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,4 +89,72 @@ To name the pre-release, set the name by adding `--prerelease <name>`
...
+&nbsp;
+
+
+# <a name="Development"></a> Development
+
+You can find a deployment utility with hardhat to easily deploy the smart contracts locally or on our L14 test network.
+
+All the deployment scripts for `base` contracts initialize the contract after deployment to the zero address for security.
+
+&nbsp;
```
Comment (Hugoo) [inline CONTRIBUTING.md:104]: why this?

File: deploy/008_deploy_base_universal_receiver_delegate.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  deployments,
+  getNamedAccounts,
+}: HardhatRuntimeEnvironment) => {
+  const { deploy } = deployments;
+  const { owner } = await getNamedAccounts();
+
+  const deployResult = await deploy("LSP1UniversalReceiverDelegateInit", {
+    from: owner,
+    log: true,
+    gasPrice: ethers.BigNumber.from("20000000000"), // in wei
```
Comment (Hugoo) [inline deploy/008_deploy_base_universal_receiver_delegate.ts:15]: You can write it like this to make it easier to read:

```suggestion
    gasPrice: ethers.BigNumber.from("20_000_000_000"), // in wei
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -87,4 +89,72 @@ To name the pre-release, set the name by adding `--prerelease <name>`
...
+> 
+> - `LSP6KeyManagerInit`: deploy a `UniversalProfileInit` + `KeyManagerInit`, as base contracts (**NB:** the Key Manager will be initialized with reference to `address(0)`).   
+> 
+> - `LSP1UniversalReceiverDelegate`: deploy a Universal Receiver Delegate contract
+> 
+> - `LSP1UniversalReceiverDelegateInit`: deploy a Universal Receiver Delegate as a base contract.
+> 
+> - `standard`: deploy the 3 standard contract above.
+
+> - `base`: deploy the 3 base contract above (for proxy use)
```
Comment (Hugoo) [inline CONTRIBUTING.md:143]: Not sure if these `>` adds value?

I think it makes the output a bit "hard to read":

image

**Review by Hugoo — APPROVED** [review]
cool !

**Comment by frozeman** [comment]
Please document in the readme, and merge if done @CJ42

File: CONTRIBUTING.md (hunk 1 above)
Comment (CJ42) [reply CONTRIBUTING.md:104]: just for visual spacing, otherwise it looks very close to the paragraph otherwise.

**Comment by CJ42** [comment]
> Please document in the readme, and merge if done @CJ42

@frozeman done ✅

**Pull request merged**